cmd/query/app/http_handler_test.go

```diff
@@ -623,6 +632,39 @@ func TestGetOperationsLegacyStorageFailure(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestTransformOTLPSuccess(t *testing.T) {
+	withTestServer(func(ts *testServer) {
+		requestBytes := readOTLPTraces(t)
+
+		resp, err := ts.server.Client().Post(ts.server.URL+"/api/transform", "application/json", bytes.NewReader(requestBytes))
+		require.NoError(t, err)
+
+		responseBytes, err := io.ReadAll(resp.Body)
+		require.NoError(t, err)
+
+		corectResponseBytes := readJaegerTraces(t)
+		assert.Equal(t, corectResponseBytes, responseBytes)
+	}, querysvc.QueryServiceOptions{})
+}
+
+func TestTransformOTLPReadError(t *testing.T) {
+	withTestServer(func(ts *testServer) {
+		bytesReader := &IoReaderMock{}
+		bytesReader.On("Read", mock.AnythingOfType("[]uint8")).Return(0, errors.New("Mocked error"))
+		_, err := ts.server.Client().Post(ts.server.URL+"/api/transform", "application/json", bytesReader)
```

Comment:
yes this will never read the logic we written but by using this mocked IO reader I'm  trying to test the error block in httphandler API to increase test percentages (which are still low as I haven't been able to cover convertToUI)

Reply:
ok. But I see coverage already very high, you don't need to add any more
```
$ go test -cover ./cmd/query/app
ok  	github.com/jaegertracing/jaeger/cmd/query/app	3.683s	coverage: 99.0% of statements
```

Reply:
Do you suggest removing this test as well?

Reply:
let's keep it